StandardLibrary/runtimes/net/STD.csproj

```diff
@@ -64,9 +64,6 @@
     <PackageReference Include="System.Collections.Immutable" Version="1.7.0" />
     <!-- Work around for dafny-lang/dafny/issues/1951; remove once resolved -->
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
-    <Compile Include="..\..\..\libraries\src\FileIO\FileIO.cs">
```

Comment:
It looks like you got your change merged into Dafny libraries: https://github.com/dafny-lang/libraries/pull/163
Do we still want to remove our dependency on that in favor of the new `FileIO.cs`?
(I could be okay with this but just want to check)

Reply:
Yes, it seemed excessively confusing to have just one extern in just one language implemented in another repo, so now all of our externs are in one place.

Reply:
Agreed